### PR TITLE
Fix typos in Clipboard docs

### PR DESCRIPTION
**Pull request by Maan2003**

### CODE REVIEW

File: druid-shell/src/clipboard.rs [hunk 1]
```diff
@@ -36,7 +36,7 @@ pub use crate::platform::clipboard as platform;
 /// data available as an SVG, for other editors, and a bitmap image for applications
 /// that can accept general image data.
 ///
-/// ## `FormatId`entifiers
+/// ## `FormatId` entifiers
```
Comment (jneem) [inline druid-shell/src/clipboard.rs:39]: Maybe this one was intentional? because, like, "FormatIdentifiers". It isn't great, though

**Review by jneem — APPROVED** [review]

File: druid-shell/src/clipboard.rs (hunk 1 above)
Comment (Maan2003) [reply druid-shell/src/clipboard.rs:39]: oh :sweat_smile: missed it. 
Any suggestions? I would make it just `FormatId` :man_shrugging:

Comment (cmyr) [reply druid-shell/src/clipboard.rs:39]: This was intentional, but maybe not helpful. I'd either leave it as it was, or I would replace it with plain `FormatId`.

Comment (Maan2003) [reply druid-shell/src/clipboard.rs]: leaving it

**Pull request merged**